app/controllers/moderators_controller.rb

```diff
@@ -7,8 +7,8 @@ def edit
 
   def index
     session[:current_moderator_id]=1
```

Comment:
Never explicitly assign the id. You could identify moderator by email. If you change DB, this will fail.